docs/03-endpoints/api-admin/projects.md

```diff
@@ -784,6 +784,7 @@ Example response:
 Permissions: ProjectAdmin
 
 Request definition:
+
```

Comment:
without empty line, mkdocs doesn't recognize this as bullet point list, and renders it very ugly. See https://docs.dasch.swiss/2023.10.02/DSP-API/03-endpoints/api-admin/projects/#set-restricted-view-settings